doc/developer/debugging.rst

```diff
@@ -107,6 +164,25 @@ Add in the ``docker-compose.yaml`` file, in the ``geoportal`` service the follow
          - <c2cgeoportal_git_root>/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal_geoportal/c2cgeoportal_geoportal
          - <c2cgeoportal_git_root>/geoportal/c2cgeoportal_admin:/opt/c2cgeoportal_geoportal/c2cgeoportal_admin
 
+Expose a service
+................
+
+To expose a service out of the docker-compose you can add a port for the service in the vars, e.g.:
```

Comment:
In my understanding: we are exposing a service out of the docker container, not out of the docker-compose?
Or are we saying here:
To expose a service from the docker container via docker-compose, you can add...

Reply:
No inside the composition we can access to non exposed port ...